src/main/java/edu/utdallas/davisbase/representation/CreateTableCommandRepresentation.java

```diff
@@ -0,0 +1,37 @@
+package edu.utdallas.davisbase.representation;
+
+import net.sf.jsqlparser.statement.create.table.ColumnDefinition;
+
+import java.util.List;
+
+public class CreateTableCommandRepresentation implements CommandRepresentation {
```

Comment:
This class is missing the following getter methods:

- `public String getTable()`
- `public List<ColumnDefinition> getColumnDefinitions`